File: packages/insomnia-httpsnippet/test/fixtures/output/ocaml/cohttp/headers.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.add_list (Header.init ()) [
  ("accept", "application/json");
  ("x-foo", "Bar");
] in

Client.call ~headers `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/jquery/headers.js
var settings = {
  async: true,
  crossDomain: true,
  url: 'http://mockbin.com/har',
  method: 'GET',
  headers: {
    accept: 'application/json',
    'x-foo': 'Bar',
  },
};

$.ajax(settings).done(function(response) {
  console.log(response);
});
